Could not find a package configuration file provided by "loader" - Build errors in SpacesArm64GeckoGeneric variant

## Configuration

<!--- State the version number and build ID affected. -->
<!--- The build ID is obtained by clicking on the build date in settings. -->
Wolvic version: 1.5.1
Wolvic build ID: 116.0

<!--- Include the name and version of the hardware/VR headset you experienced the bug in. -->
Hardware: Digilens ARGO

## Context
I’m trying to build a version of the Wolvic XR Browser integrated with Snapdragon Spaces for our ARGO smart glasses platform. I am facing some builds errors when I'm trying to build project in the android studio.
1. I followed the developer workflow from (https://github.com/Igalia/wolvic/wiki/Developer-workflow#building-gecko)
2. I was able to successfully build Gecko and add Gecko dependency to my android project.
3. Active Build Variant:  SpacesArm64GeckoGenericDebug, Active ABI:  arm64-v8a
4. I have libopenxr_loader.aar and qxrclients.aar files under wolvic/third_party/snapdragon-spaces/ for the Snapdragon Spaces integration.

## Possible Solution
The cmake for including Snapdragon Spaces libraries is missing.
 
## Error Logs and Stack Traces
<details open>

## Build Output - START
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:configureCMakeDebug[arm64-v8a]-7'.
> [CXX1429] error when building with cmake using /home/sujay.anil/Sujay/wolvic/app/CMakeLists.txt: -- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
  -- Android: Selected unified Clang toolchain
  -- The C compiler identification is Clang 12.0.8
  -- The CXX compiler identification is Clang 12.0.8
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - done
  -- Check for working C compiler: /home/sujay.anil/Android/Sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/clang - skipped
  -- Detecting C compile features
  -- Detecting C compile features - done
  -- Detecting CXX compiler ABI info
  -- Detecting CXX compiler ABI info - done
  -- Check for working CXX compiler: /home/sujay.anil/Android/Sdk/ndk/23.1.7779620/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++ - skipped
  -- Detecting CXX compile features
  -- Detecting CXX compile features - done
  -- Configuring incomplete, errors occurred!
  See also "/home/sujay.anil/Sujay/wolvic/app/.cxx/Debug/3f424t71/arm64-v8a/CMakeFiles/CMakeOutput.log".
  
  C++ build system [configure] failed while executing:
      /home/sujay.anil/Android/Sdk/cmake/3.22.1/bin/cmake \
        -H/home/sujay.anil/Sujay/wolvic/app \
        -DCMAKE_SYSTEM_NAME=Android \
        -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
        -DCMAKE_SYSTEM_VERSION=24 \
        -DANDROID_PLATFORM=android-24 \
        -DANDROID_ABI=arm64-v8a \
        -DCMAKE_ANDROID_ARCH_ABI=arm64-v8a \
        -DANDROID_NDK=/home/sujay.anil/Android/Sdk/ndk/23.1.7779620 \
        -DCMAKE_ANDROID_NDK=/home/sujay.anil/Android/Sdk/ndk/23.1.7779620 \
        -DCMAKE_TOOLCHAIN_FILE=/home/sujay.anil/Android/Sdk/ndk/23.1.7779620/build/cmake/android.toolchain.cmake \
        -DCMAKE_MAKE_PROGRAM=/home/sujay.anil/Android/Sdk/cmake/3.22.1/bin/ninja \
        "-DCMAKE_CXX_FLAGS=-std=c++14 -fexceptions -frtti -Werror -I/home/sujay.anil/Sujay/wolvic/app/src/main/cpp -I/home/sujay.anil/Sujay/wolvic/app/src/main/cpp/vrb/include -DANDROID  -DGECKO  -DSPACES -DOPENXR -I/home/sujay.anil/Sujay/wolvic/app/src/openxr/cpp" \
        -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=/home/sujay.anil/Sujay/wolvic/app/build/intermediates/cxx/Debug/3f424t71/obj/arm64-v8a \
        -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=/home/sujay.anil/Sujay/wolvic/app/build/intermediates/cxx/Debug/3f424t71/obj/arm64-v8a \
        -DCMAKE_BUILD_TYPE=Debug \
        -DCMAKE_FIND_ROOT_PATH=/home/sujay.anil/Sujay/wolvic/app/.cxx/Debug/3f424t71/prefab/arm64-v8a/prefab \
        -B/home/sujay.anil/Sujay/wolvic/app/.cxx/Debug/3f424t71/arm64-v8a \
        -GNinja \
        -DANDROID_STL=c++_shared \
        -DGECKO=ON \
        -DSPACES=ON \
        -DOPENXR=ON
    from /home/sujay.anil/Sujay/wolvic/app
  CMake Error at CMakeLists.txt:203 (find_package):
    Could not find a package configuration file provided by "loader" with any
    of the following names:
  
      loaderConfig.cmake
      loader-config.cmake
  
    Add the installation prefix of "loader" to CMAKE_PREFIX_PATH or set
    "loader_DIR" to a directory containing one of the above files.  If "loader"
    provides a separate development package or SDK, be sure it has been
    installed.
## Build Output - END

<!--- DO NOT REMOVE THIS LINE. -->
```
```
<!--- DO NOT REMOVE THIS LINE. -->

</details>

--- a/README.md
+++ b/README.md
@@ -51,7 +51,7 @@ This repo is only available to Igalia members. If you have access to the relevan
  - `third_party/wavesdk/` for Vive (should contain a `build` folder, among other things)
  - `third_party/picoxr` [Pico OpenXR Mobile SDK](https://developer-global.pico-interactive.com/sdk?deviceId=1&platformId=3&itemId=11) (should contain `include` and `libs` folders, among other things that are not necessary for Wolvic)
  - `third_party/lynx` [for Lynx](https://portal.lynx-r.com)(should contain a `loader-release.aar` file)
- - `third_party/snapdragon-spaces` [for Snapdragon Spaces](https://spaces.qualcomm.com/)(should contain `libopenxr_loader.aar` and `qxrclients.aar` files)
+ - `third_party/spaces` [for Snapdragon Spaces](https://spaces.qualcomm.com/)(should contain `libopenxr_loader.aar`)
  - `third_party/OpenXR-SDK/` [OpenXR-SDK](https://github.com/KhronosGroup/OpenXR-SDK) (should contain an `include` folder)
  - `third_party/aliceimu/` for [Huawei Vision Glass](https://consumer.huawei.com/cn/wearables/vision-glass/) (should contain an `.aar` file with the IMU library for the glasses)
 

